package.json

```diff
@@ -118,18 +118,19 @@
     "lint-fix": "eslint --fix --cache ./",
     "build": "babel src/ -d lib/ --copy-files",
     "watch": "babel --watch src/ -d lib/ --copy-files",
-    "test:mongodb:runnerstart": "cross-env MONGODB_VERSION=${MONGODB_VERSION:=$npm_config_dbversion} MONGODB_TOPOLOGY=${MONGODB_TOPOLOGY:=standalone} mongodb-runner start",
+    "test:mongodb:runnerstart": "cross-env mongodb-runner stop --all && mongodb-runner start -t standalone --version $npm_config_dbversion -- --port 27017",
```

Comment:
- Keep the env var fallback `MONGODB_VERSION=${MONGODB_VERSION:=$npm_config_dbversion}`.
- Remove the `mongodb-runner stop --all` from the start command, it's already in `posttest`.